source/blueprints/user/services.py

```diff
@@ -130,4 +131,23 @@ def verify_password(current_password: str, new_password: str, confirm_password:
     if bcrypt.checkpw(new_password.encode("utf-8"), current_user.password):
         raise ValueError("Password is the same as the old password")
 
-    return hash_password(new_password)
\ No newline at end of file
+    return hash_password(new_password)
+
+
+def delete_user(answer: str):
+    """Save the date of delete account at column 'deleted_at' for User, Profile and Game Stats."""
+    
+    if answer == "yes":
```

Comment:
The logic of receiving a "confirm_deletion" flag is correct, but we don't need to have it in the backend, in this context this is a concern of the frontend, so we can securely remove it here.